Userland/Libraries/LibGUI/AboutDialog.cpp

```diff
@@ -24,8 +24,8 @@ NonnullRefPtr<AboutDialog> AboutDialog::create(String const& name, String versio
     auto dialog = adopt_ref(*new AboutDialog(name, version, icon, parent_window));
     dialog->set_title(ByteString::formatted("About {}", name));
 
-    auto widget = dialog->set_main_widget<Widget>();
-    MUST(widget->load_from_gml(about_dialog_gml));
+    auto widget = MUST(AboutDialogWidget::try_create());
```

Comment:
This should most likely be propagating errors (or `.release_value_but_fixme_should_propagate_errors()`) instead of assuming there are none.